components/navbar.vue

```diff
@@ -9,6 +9,9 @@
           <li class="font-semibold text-white">
             <nuxt-link to="/">Tasks</nuxt-link>
           </li>
+          <li>
+            <img :src="'https://ui-avatars.com/api/?name=' + name + '&background=330066&size=33&color=fff'" class="rounded-circle" :id="avatar-image"><span class="todo-profile-name" id="profile-name" >{{name}}</span>
```

Comment:
id is bound to avatar-image even though there is no such data/computed property.

Reply:
removed the v-bind